test/controllers/property.controller.test.js

```diff
@@ -1275,7 +1280,7 @@ describe('Property Controller', () => {
       {
         userId: regularUser._id,
         propertyId: property._id,
-        offerId: offer._id,
+        model: { id: offer._id, type: VALID_TRANSACTION_MODELS.OFFER },
```

Comment:
Using `MODEL.OFFER` is better here to ensure consistency . Please update for other ones too.

```suggestion
model: { id: offer._id, type: MODELS.OFFER },
```